src/operation/get-requestbody-examples.js

```diff
@@ -1,30 +1,4 @@
-const { getMediaTypeExample, getMediaTypeExamples } = require('../lib/get-mediatype-examples');
-const cleanStringify = require('../lib/json-stringify-clean');
-
-/**
- * Construct an object for a media type and any examples that its Media Type Object might hold.
- *
- * This code is identical to `get-response-examples` except that this returns the media type as `mediaType` instead of
- * `language`.
- *
- * @link https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#mediaTypeObject
- * @param {string} mediaType
- * @param {object} mediaTypeObject
- * @param {(object|false)} example
- * @returns {(object|false)}
- */
-function constructMediaType(mediaType, mediaTypeObject, example) {
```

Comment:
This `constructMediaType` method in the response and request body example generators was similar enough that I was able to consolidate all of this logic into a single `getMediaTypeExamples` library that can handle both as they're both now just OAS [Media Type Objects](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#mediaTypeObject).